demo-frontend/Types.elm

```diff
@@ -0,0 +1,31 @@
+module Types exposing (..)
+
+import Http
+import File exposing (File)
+
+type alias Color = String
+type alias ImageString = String
+type Algorithm = MedianCut | Kmeans | KmeansPP
+
+algorithmToString : Algorithm -> String
```

Comment:
to chyba nie tutaj powinno się znaleźć 😉 util albo jakis l10n?

Reply:
poprawione